ictv/models/user.py

```diff
@@ -41,7 +42,7 @@ class User(ICTVObject):
     password = StringCol(default=None)  # Used for local login
     reset_secret = StringCol(notNone=True)  # Used for local login to reset password
     has_toured = BoolCol(default=False)  # Has the user completed the app tour
-    creation_date = DateTimeCol(default=None)
+    creation_date = DateTimeCol(default=datetime.now())
```

Comment:
Don't execute the function, just provide a reference so SQLObject can do the trick
http://sqlobject.org/SQLObject.html#col-class-specifying-columns, see `default`